gitlabform/processors/project/project_processor.py

```diff
@@ -20,15 +20,15 @@ def _process_configuration(self, project_and_group: str, configuration: dict):
 
             # Check if the project was already transfered (i.e. in previous run) or a project with same path already exists
             try:
-                project_in_config: Project = self.gl.projects.get(
+                project_in_config: Project = self.gl.get_project_by_name(
                     project_path_with_namespace
                 )
```

Comment:
Looking at the usage/code here, wondering if it might be more accurate to rename `get_project_by_name` to `get_project_by_path`. Technically we can have 2 projects with same name in GitLab but the path must be unique. If that make sense, maybe same for `get_group_by_name`?